.gitignore

```diff
@@ -266,4 +266,5 @@ __pycache__/
 
 # /Build/
 /Build/logs
-/Build/out
\ No newline at end of file
+/Build/out
+AnnoDesigner/logs/
```

Comment:
This folder should not exist. Maybe some leftover of some weird debuggind issue?
The logs should be under `AnnoDesigner\bin` (which is excluded).

Reply:
I think I must be building/running it differently to you. I'm just using `dotnet run` from the AnnoDesigner directory. Trying that from the root directory gives me an error but I suspect that is why my logs are going somewhere else.

Reply:
`dotnet run` does not set the working directory of the application to `AnnoDesigner/bin/[Debug|Release]`, its sets the working directory to the root of the csproj file, hence why you are seeing different behaviour.  We should probably leave these entries in the `gitignore` file. I would add a comment mentioning the above.